### DISCUSSION TITLE
About the training data of example model

### DISCUSSION THREAD

**Discussion by jacksoncsy** [Nicholasli1995/EvoSkeleton#46] [Q&A] [answered]
Hello Nicholas,

In continuation with our previous discussion, I really wonder which data and what setting do you use to train the **example_model.th** in [link](https://drive.google.com/file/d/158oCTK-9Y8Bl9qxidoHcXfqfeeA7qT93/view). 

Because I tried to use the pretrained model **model.th** provided in [link](https://drive.google.com/drive/folders/1IRKUWrnheD03Dj30LLGlh_LLT1CK6dr5?usp=sharing), which is supposed to be trained on synthetic Human3.6M, correct? Unfortunately, using this model, I can't get the same almost-perfect predictions on U3DPW as those fitted by the inference model **example_model.th** ... 

Please do help me clarify this, as it is quite important to know which model is trained on what, for the sake of a fair comparison.

I did appreciate your answers to my previous questions!

Thanks a lot,
Shiyang

**Comment by Nicholasli1995** [comment, marked as answer]
Hi, both are trained with synthetic data.

The pre-trained model S15678 was trained with synthetic data and norm_single=False. This model is intended for in-door evaluation on H36M.

The inference model (example_model.th) was trained with synthetic data and with norm_single=True. Note that the normalization method must match for both training and inference.

> Hello Nicholas,
> 
> In continuation with our previous discussion, I really wonder which data and what setting do you use to train the **example_model.th** in [link](https://drive.google.com/file/d/158oCTK-9Y8Bl9qxidoHcXfqfeeA7qT93/view).
> 
> Because I tried to use the pretrained model **model.th** provided in [link](https://drive.google.com/drive/folders/1IRKUWrnheD03Dj30LLGlh_LLT1CK6dr5?usp=sharing), which is supposed to be trained on synthetic Human3.6M, correct? Unfortunately, using this model, I can't get the same almost-perfect predictions on U3DPW as those fitted by the inference model **example_model.th** ...
> 
> Please do help me clarify this, as it is quite important to know which model is trained on what, for the sake of a fair comparison.
> 
> I did appreciate your answers to my previous questions!
> 
> Thanks a lot,
> Shiyang

**Reply by Nicholasli1995** [reply]
For reasons why two normalization methods are used, please see https://github.com/Nicholasli1995/EvoSkeleton/issues/12#issuecomment-671898325.

For in-the-wild images, the location of the subjects carry little information of their 3D poses. Thus it is OK to make the input translational-invariant. For H36M, the default normalization leads to better performance. This means that subject locations are useful for inferring their 3D poses in H36M (e.g., a person is at a particular location is informative for inferring its pose). However, this is actually dataset bias and does not generalize to unconstrained images.

> Hello Nicholas,
> 
> In continuation with our previous discussion, I really wonder which data and what setting do you use to train the **example_model.th** in [link](https://drive.google.com/file/d/158oCTK-9Y8Bl9qxidoHcXfqfeeA7qT93/view).
> 
> Because I tried to use the pretrained model **model.th** provided in [link](https://drive.google.com/drive/folders/1IRKUWrnheD03Dj30LLGlh_LLT1CK6dr5?usp=sharing), which is supposed to be trained on synthetic Human3.6M, correct? Unfortunately, using this model, I can't get the same almost-perfect predictions on U3DPW as those fitted by the inference model **example_model.th** ...
> 
> Please do help me clarify this, as it is quite important to know which model is trained on what, for the sake of a fair comparison.
> 
> I did appreciate your answers to my previous questions!
> 
> Thanks a lot,
> Shiyang

**Comment by jacksoncsy** [comment]
Hello Nicholas,

Sorry for a late response! I will try them and let you know, thanks a lot for those information!

Best,
Shiyang
